app/components/Form/ProjectFundingAgreementForm.tsx

```diff
@@ -424,11 +422,7 @@ const ProjectFundingAgreementForm: React.FC<Props> = (props) => {
                       size="small"
                       className="removeButton"
                       onClick={() =>
-                        deleteAdditionalFundingSource(
-                          formChange.id,
-                          formChange.rowId,
-                          formChange.operation
-                        )
+                        deleteAdditionalFundingSource(index + 1, formChange.id)
```

Comment:
deleting things by this generated index is going to cause a bug.
Scenario:
create 3 sources:
source 1 generated index 1. db source_index: 1
source 2 generated index 2, db source_index: 2
source 3 generated index 3, db_source_index: 3

delete source 2
you now have 
source 1 generated index 1, db_source_index 1
source 3 generated index 2, db_source_index 3 <- The discard will no longer be able to delete this one as the front-end generated index and the index saved in the db no longer match